tests/kafkatest/services/security/security_config.py

```diff
@@ -166,10 +167,22 @@ def has_sasl_kerberos(self):
 
     @property
     def kafka_opts(self):
+        if SecurityConfig.ZK_JAAS_LOGIN:
+            return "\"-Djava.security.auth.login.config=%s\"" % SecurityConfig.ZK_JAAS_LOGIN
         if self.has_sasl:
             return "\"-Djava.security.auth.login.config=%s -Djava.security.krb5.conf=%s\"" % (SecurityConfig.JAAS_CONF_PATH, SecurityConfig.KRB5CONF_PATH)
-        else:
-            return ""
+        return ""
+
+    def gen_jaas_login_digest(self, nodes):
+        jaas_file = self.render('zk_jaas_digest.login')
+        for node in nodes:
+            node.account.ssh("mkdir -p %s" % SecurityConfig.CONFIG_DIR, allow_fail=False)
+            node.account.create_file("%s/zk_jaas_digest.login" % SecurityConfig.CONFIG_DIR, jaas_file)
+
+        return "%s/zk_jaas_digest.login" % SecurityConfig.CONFIG_DIR
+
+    def set_zk_jaas_login(self, login):
```

Comment:
Similar to comment in `ZookeeperService` above
Use an instance attribute instead of class attribute for `zk_jaas_login`, and let external users update it directly